python/tests/graph/test_utils.py

```diff
@@ -0,0 +1,92 @@
+# Copyright 2017 Databricks, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+from __future__ import absolute_import, division, print_function
+
+from collections import namedtuple
+# Use this to create parameterized test cases
+from parameterized import parameterized
+
+import tensorflow as tf
+
+import sparkdl.graph.utils as tfx
+
+from ..tests import PythonUnitTestCase
+
+TestCase = namedtuple('TestCase', ['data', 'description'])
+
+
+def _gen_graph_elems_names():
+    op_name = 'someOp'
+    for tnsr_idx in range(17):
+        tnsr_name = '{}:{}'.format(op_name, tnsr_idx)
+        yield TestCase(data=(op_name, tfx.op_name(tnsr_name)),
+                       description='must get the same op name from its tensor')
+        yield TestCase(data=(tnsr_name, tfx.tensor_name(tnsr_name)),
+                       description='must get the tensor name from its operation')
+
+
+def _gen_wrong_graph_elems_types():
+    for wrong_val in [7, 1.2, tf.Graph()]:
+        yield TestCase(data=wrong_val, description='wrong type {}'.format(type(wrong_val)))
+
+
+def _gen_graph_elems():
+    op_name = 'someConstOp'
+    tnsr_name = '{}:0'.format(op_name)
+    tnsr = tf.constant(1427.08, name=op_name)
+    graph = tnsr.graph
+
+    yield TestCase(data=(op_name, tfx.op_name(tnsr)),
```

Comment:
since there are so many test cases here, let's add comments in between sections, e.g. 
```
# Tests for op_name
yield ..
...
# Tests for tensor_name
...
# Tests for get_op
...
# Tests for get_tensor
...
# ...
```